helm/hocs-outbound-proxy.yaml

```diff
@@ -7,3 +7,7 @@ app:
   resources:
     requests:
       cpu: 50m
+
+
+rdsCIDR: {{ env "RDS_CIDR" }}
```

Comment:
I do feel this could probably have a better layout than at root of the yml file. Not blocking though...